server/screens.js

```diff
@@ -33,6 +34,58 @@ function generateAdminUpdate(sock) {
 	});
 }
 
+function decideWhichScreenGetsToKeepAnID(screenA, screenB){
+
+	// Emit an event to the screens to reassign using the ID and the timestamp 
+	// of when that id was assigned as the identifier for the screen that needs
+	// to reassign. The original screen (and the rest) can ignore this message.
+	const screenToChange = screenA.idUpdated > screenB.idUpdated ? screenA : screenB ;
+	app.io.of('/screens').emit('reassign', { id : screenToChange.id, idUpdated : screenToChange.idUpdated, newID : generateID() });
+
+}
+
+
+function checkForConflictingId(id){
+
+	return assignedIDs.some(existingScreen => {
+		return existingScreen.id == id;
+	});
+
+}
+
+function checkForConflictingScreens(data){
+
+	return assignedIDs.some(existingScreen => {
+
+		if(existingScreen.id === data.id){
+			// Screen has a matching id
+			if(existingScreen.idUpdated === data.idUpdated){
+				// Screen is the same screen as the one it's checking against 
+				return false;
+			} else {
+				/// Screen is different, there is a conflict
+				return true;
+			}
+
+		} else {
+			return false;
+		}
+
+	});
+
+}
+
+function generateID(){
+
+	var newID = parseInt(Math.random() * 99999 | 0, 10);
```

Comment:
It rounds down to the nearest whole number. We can use Math.floor() if you prefer?